app/Http/Controllers/Bible/BibleVersesController.php

```diff
@@ -75,7 +75,15 @@ public function showVerseByLanguage(
         $page           = checkParam('page') ?? 1;
 
         $access_group_ids = getAccessGroups();
-        $cache_params = [$limit, $page, $language_code, $this->key, $book_id, $chapter_id, $verse_number];
+        $cache_params = [
+            $limit,
+            $page,
+            $language_code,
+            $access_group_ids->implode(','),
```

Comment:
@bradflood I have created a new class called `AccessGroupsCollection` that will be responsible for managing the list of access groups and this class extends the Collection class. So, this new class has a new method named: toString() where it will return the access groups list as a single string delimited by |. See the following line: https://github.com/faithcomesbyhearing/dbp/pull/917/files#diff-386a1d7987312c791039f43ba8d255517a345fd764e121d848c60ccd16db07abR1